kuma/wiki/models.py

```diff
@@ -1805,6 +1756,14 @@ def get_section_content(self, section_id):
         """Convenience method to extract the content for a single section"""
         return self.document.extract_section(self.content, section_id)
 
+    def get_tidied_content(self):
+        if self.tidied_content:
+            return self.tidied_content
+        else:
+            from .tasks import tidy_revision_content
+            tidy_revision_content.delay(self.pk)
+            return tidy_content(self.content)
```

Comment:
Yeah, absolutely, this is a compromise to be able to continue to show the diff for revisions without denormalized content now as it's the main feature of the view. It's basically only there for the time in which not all of the revisions have been tidied. Alternatively we could run a full tidying manually via Celery.